src/test_validator.py

```diff
@@ -0,0 +1,180 @@
+import unittest
+import datetime as dt
+from validation import String, Date
+
+def broken_function():
```

Comment:
Not sure why you've added this but it seems unnecessary. You can remove this function.